_includes/payment-order-other-features.md

```diff
@@ -341,19 +351,19 @@ A list of possible operations and their explanation is given below.
             "rel": "view-paymentorder",
             "contentType": "application/javascript"
         },
-        {
+        {   `DEPRECATED DO NOT USE!`
```

Comment:
`create-paymentorder-capture` is not deprecated. It is `create-capture` (which is missing from the operations list) that is deprecated.

```suggestion
        {
```